spec/hashTable.js

```diff
@@ -0,0 +1,48 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/hashTable'
+
+chai.use(chaiChange)
+
+describe.only('hashTable', () => {
+  it('exists', () => {
+    expect(HashTable).to.be.a('function')
+  })
+  context('put()', () => {
+    it('adds a key-value pair to the hash table.', () => {
+      const ht = new HashTable()
+      ht.put('name', 'frankenstein')
+      expect(ht.size()).to.equal(1)
+      ht.put('19.0', 'stein')
+      expect(ht.size()).to.equal(2)
+      ht.get('19.0')
+      expect(ht.get('19.0')).to.eql('stein')
+    })
+  })
+  context('contains()', () => {
+    it('returns true if the hash table contains the key.', () => {
+      const ht = new HashTable()
+      expect(ht.contains('name')).to.be.false
+      ht.put('name', 'frankenstein')
+      ht.put('19.0', 'stein')
+      expect(ht.contains('name')).to.be.true
+    })
+  })
+  context('remove()', () => {
+    it('removes a key-value pair by key.', () => {
```

Comment:
I'm sure you're on your way to this, but:
```js
  context('remove()', () => {
    it('removes a key-value pair by key.', () => {
      const ht = new HashTable()
      ht.put('name', 'frankenstein')
      expect(ht.contains('name')).to.be.true
      ht.remove('name')
      expect(ht.contains('name')).to.be.false
    })
  })
```

And then test that size decreases.